config/bundles.php

```diff
@@ -5,26 +5,23 @@
     Symfony\Bundle\MonologBundle\MonologBundle::class => ['all' => true],
     Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle::class => ['all' => true],
     Symfony\Bundle\TwigBundle\TwigBundle::class => ['all' => true],
-    Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
     Symfony\Bundle\DebugBundle\DebugBundle::class => ['dev' => true, 'test' => true],
     Enqueue\Bundle\EnqueueBundle::class => ['all' => true],
     Enqueue\MessengerAdapter\Bundle\EnqueueAdapterBundle::class => ['all' => true],
     Shopware\Core\Framework\Framework::class => ['all' => true],
     Shopware\Core\System\System::class => ['all' => true],
     Shopware\Core\Content\Content::class => ['all' => true],
     Shopware\Core\Checkout\Checkout::class => ['all' => true],
-    Shopware\Core\Profiling\Profiling::class => ['dev' => true],
+    Shopware\Core\Maintenance\Maintenance::class => ['all' => true],
```

Comment:
the composer version specifies that `"shopware/core": "~v6.4.0"` is required, so we can't expect that the Maintenance and DevOps bundles are present in all installations i guess. 

Please bring back the class exists checks for both cases, maybe you can add a `@deprecated tag:v6.5.0` annotation so we will remove the class exists checks on the next major version

Reply:
Done. Thank you.